runelite-client/src/main/java/net/runelite/client/game/AgilityShortcut.java

```diff
@@ -65,7 +65,9 @@
 			// Asgarnian Ice Dungeon
 			STEPS,
 			// Fossil Island Wyvern Cave
-			STAIRS_31485),
+			STAIRS_31485,
```

Comment:
Remove whitespace after ','

Reply:
I'm not sure if I understood correctly but I think that's what you wanted changed right?

Reply:
Yea, just to pass the tests